### PR TITLE
Feature/fix place profile ui

### PR DESCRIPTION
**Pull request by stasberkman**

### CODE REVIEW

File: Dizzy/Screens/PlaceProfile/PlaceProfileVM.swift [hunk 1]
```diff
@@ -30,19 +31,41 @@ final class PlaceProfileVM: PlaceProfileVMType {
...
     
-    func whatsappToPublicistPressed() {
+    func requestTableButtonPressed() {
         let whatsappText = "Hi I want to order a table".localized.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed)
         guard let phoneNumber = placeInfo.publicistPhoneNumber, !phoneNumber.isEmpty,
             let url = URL(string: "https://wa.me/\(placeInfo.publicistPhoneNumber ?? "")/?text=\(whatsappText ??    "")") else { return }
         UIApplication.shared.open(url, options: [:])
     }
     
+    func openWaze(location : Location) {
```
Comment (menesh2) [inline Dizzy/Screens/PlaceProfile/PlaceProfileVM.swift]: move it to separate entity that we could use it in other places if we would like

File: Dizzy/Screens/Discovery/Cells/DiscoveryPlaceCell.swift [hunk 2]
```diff
@@ -23,7 +23,7 @@ class DiscoveryPlaceCell: UICollectionViewCell {
     
     let stackViewTrailingPadding = CGFloat(15)
```
Comment (menesh2) [inline Dizzy/Screens/Discovery/Cells/DiscoveryPlaceCell.swift:24]: on real device, the press on place text is not good because other place gets the press

**Review by menesh2 — CHANGES_REQUESTED** [review]

File: Dizzy/Screens/PlaceProfile/PlaceProfileVM.swift (hunk 1 above)
Comment (stasberkman) [reply Dizzy/Screens/PlaceProfile/PlaceProfileVM.swift]: Done - see ExternalNavigationProvider

File: Dizzy/Screens/Discovery/Cells/DiscoveryPlaceCell.swift (hunk 2 above)
Comment (stasberkman) [reply Dizzy/Screens/Discovery/Cells/DiscoveryPlaceCell.swift:24]: Works as charm!

File: Dizzy/Entities/ExternalNavigationProvider.swift [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+class ExternalNavigationProvider {
+
+    func openWaze(location : Location) {
+        if let url = URL(string: "waze://"), UIApplication.shared.canOpenURL(url) {
+            let urlStr: String = "waze://?ll=\(location.latitude),\(location.longitude)&navigate=yes"
+            if let url = URL(string: urlStr) {
+                UIApplication.shared.open(url, options: [:], completionHandler: nil)
+            }
+        } else {
+            // Waze is not installed. Launch AppStore to install Waze app
```
Comment (menesh2) [inline Dizzy/Entities/ExternalNavigationProvider.swift:20]: instead of writing mark, please move this code in to function called `askToInstallWaze`

**Review by menesh2 — CHANGES_REQUESTED** [review]

**Pull request merged**